xds/src/test/java/io/grpc/xds/FilterChainMatchingProtocolNegotiatorsTest.java

```diff
@@ -811,8 +825,7 @@ filterChainLessSpecific, randomConfig("no-match")),
   }
 
   @Test
-  public void sourcePrefixRange_2Matchers_expectException()
-          throws UnknownHostException {
+  public void sourcePrefixRange_2Matchers_expectException() throws IllegalArgumentException {
```

Comment:
`throws IllegalArgumentException` doesn't do anything because it is a RuntimeException.